NOTICE

```diff
@@ -0,0 +1,127 @@
+Copyright (c) 2024 xDEM developers.
+
+This file is part of xDEM (see https://github.com/GlacioHack/xdem).
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+     http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+xDEM software is distributed under the Apache Software License (ASL) v2.0, see
+LICENSE file or http://www.apache.org/licenses/LICENSE-2.0 for details.
+
+Python: programming language that lets you work quickly and integrate systems more effectively.
+Copyright (c) 2001-2023 Python Software Foundation. All Rights reserved.
```

Comment:
Just out of curiosity, will we have to update the dates annually in this file? @adebardo